### PR TITLE
Implemented IsBalanced function

### PR DESCRIPTION
**Pull request by cyberdr0id**

### CODE REVIEW

**Review by roflanKisel — CHANGES_REQUESTED** [review]
Good job!

Please, take a look at the following things: 
- Add golangci-lint config to repo and fix all the issues it reports
- Add name to test cases, add tests for edge cases
- Add test helper to assert result

File: brackets/brackets_test.go [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+	for _, tc := range testTable {
+		t.Run(tc.testName, func(t *testing.T) {
+			assertBalance(t, tc.input, tc.expected)
+		})
+	}
+}
+
+func assertBalance(t *testing.T, s string, want bool) {
+	t.Helper()
+	got := IsBalanced(s)
```
Comment (roflanKisel) [inline brackets/brackets_test.go:62]: Assert function must only compare expected result and actual result
Invocation of function, which we are testing, must be inside of test function

File: brackets/brackets_test.go [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+	if got != want {
+		t.Errorf("brackets.IsBalanced(\"%v\"): got %v, want %v", s, got, want)
```
Comment (roflanKisel) [inline brackets/brackets_test.go:65]: Assert function name and error message should be more generic: it mustn't reference IsBalanced method name at all. I.e. if we add one more test that requires bool comparison we'll be able to use the same assert function to compare bools

**Review by roflanKisel — CHANGES_REQUESTED** [review]
Great!

Please, fix comments and add README file.

**Review by roflanKisel — APPROVED** [review]
